build_test/lib/src/resolve_source.dart

```diff
@@ -118,13 +118,13 @@ Future<T> resolveSource<T>(
 /// - by default, [PackageAssetReader.currentIsolate]. A custom [packageConfig]
 /// may be provided to map files not visible to the current package's runtime.
 Future<T> resolveSources<T>(
-  Map<String, String> inputs,
+  Map<String, String>? inputs,
```

Comment:
Lets go ahead and make this non-nullable - it was already doing a null check right away :).

You can use `ArgumentError.checkNotNull` to retain the check for legacy codebase interactions, if desired (but I wouldn't bother)